src/main/kotlin/com/demonwav/mcdev/nbt/lang/psi/mixins/impl/NbttTagNameImplMixin.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.nbt.lang.psi.mixins.impl
+
+import com.demonwav.mcdev.nbt.lang.psi.mixins.NbttTagNameMixin
+import com.intellij.extapi.psi.ASTWrapperPsiElement
+import com.intellij.lang.ASTNode
+
+abstract class NbttTagNameImplMixin(node: ASTNode) : ASTWrapperPsiElement(node), NbttTagNameMixin {
+
+    override fun getTagName(): String {
+        val noQuotes = if (text.startsWith("\"")) {
```

Comment:
This looks like the same as in `NbttStringImplMixin`. Can we move this to a shared method?